client/web/compose/src/components/PageBlocks/RecordListBase.vue

```diff
@@ -124,29 +122,72 @@
         </b-row>
 
         <div
-          v-if="activeFilters.length || drillDownFilter || options.showDeletedRecordsOption"
-          class="d-flex"
+          v-if="drillDownFilter || options.showDeletedRecordsOption || groupRecordListFilter.length"
+          class="d-block gap-1"
         >
           <div
-            v-if="activeFilters.length"
-            class="d-flex align-items-center flex-wrap"
+            v-if="groupRecordListFilter.length"
+            class="d-flex flex-wrap gap-2"
           >
-            {{ $t('recordList.filter.filters.active') }}
-            <b-form-tags
-              size="lg"
-              class="d-flex align-items-center border-0 p-0 bg-transparent"
-              style="width: fit-content;"
+            <div
+              v-for="(filterGroup, groupIdx) in groupRecordListFilter"
+              :key="groupIdx"
+              class="d-flex align-items-center gap-2"
             >
-              <b-form-tag
-                v-for="(title, i) in activeFilters"
-                :key="i"
-                :title="title"
-                variant="light"
-                pill
-                class="align-items-center ml-2"
-                @remove="removeFilter(i)"
-              />
-            </b-form-tags>
+              <div class="d-flex align-items-center border rounded p-1 gap-1 flex-wrap">
+                <div
+                  v-for="(f, filterIndex) in filterGroup.filter"
+                  :key="filterIndex"
+                  class="d-flex align-items-center gap-1 rounded px-2 py-1 text-nowrap bg-light"
+                >
+                  <span>
+                    {{ formatActiveFilterName(f) }}
```

Comment:
Since you have gap 1 you have to have each text in its own element
Otherwise the gap is uneven.